src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/TokenValidatedContext.cs

```diff
@@ -8,16 +10,22 @@
 
 namespace Microsoft.AspNetCore.Authentication.OpenIdConnect
 {
-    public class AuthenticationValidatedContext : BaseOpenIdConnectContext
+    public class TokenValidatedContext : BaseOpenIdConnectContext
     {
-        public AuthenticationValidatedContext(HttpContext context, OpenIdConnectOptions options, AuthenticationProperties properties)
+        /// <summary>
+        /// Creates a <see cref="TokenValidatedContext"/>
+        /// </summary>
+        public TokenValidatedContext(HttpContext context, OpenIdConnectOptions options)
             : base(context, options)
         {
-            Properties = properties;
         }
 
-        public AuthenticationProperties Properties { get; }
+        public AuthenticationProperties Properties { get; set; }
+        
+        public JwtSecurityToken JwtSecurityToken { get; set; }
```

Comment:
We use `Token` to refer to the raw `string` elsewhere. How about `ValidatedToken`?